teuthology/orchestra/remote.py

```diff
@@ -276,11 +277,20 @@ def write_file(self, path, data, sudo=False, mode=None, owner=None,
         :param owner:   set file owner if provided
         :param mkdir:   preliminary create the file directory, defaults False
         :param append:  append data to the file, defaults False
+        :param bs:      write up to N bytes at a time, defaults 512
```

Comment:
I guess default is None, maybe just give a reference to `dd`, just add another line right after `Write data to remote file` above, like:

```The data written in 512-byte blocks, provide `bs` to use bigger blocks.```